src/portal/src/app/shared/shared.utils.ts

```diff
@@ -75,4 +75,30 @@ export const maintainUrlQueryParmas = function (uri: string, key: string, value:
         }
     }
 };
+/**
+  * the password or secret must longer than 8 chars with at least 1 uppercase letter, 1 lowercase letter and 1 number
+  * @param randomFlag
+  * @param min
+  * @param max
+  * @returns {string}
+  */
+ export function randomWord(randomFlag, min, max) {
+    let str = "0aA",
```

Comment:
"the password or secret must longer than 8 chars with at least 1 uppercase letter, 1 lowercase letter and 1 number":  all the random password should begin with "0aA"? if not,  this function can be improved.